### PR TITLE
Bring back subsite in the CWP kitchen sink

### PR DESCRIPTION
**Pull request by maxime-rainville**
I accidentally remove subsite from the sink while merging up other PRs.

### CODE REVIEW

File: .cow.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
   "upgrade-only": [
     "dnadesign/silverstripe-elemental-subsites",
     "dnadesign/silverstripe-elemental-userforms",
-    "undefinedoffset/sortablegridfield"
+    "undefinedoffset/sortablegridfield",
+    "silverstripe/subsites"
```
Comment (maxime-rainville) [inline .cow.json:23]: Subsite is now meant to be an upgrade only module.

Comment (dnsl48) [reply .cow.json:23]: perhaps we should make it so in the `.cow.json` config then?

Comment (maxime-rainville) [reply .cow.json:23]: Not sure I understand ... isn't what I'm doing right here?

Comment (dnsl48) [reply .cow.json:23]: I'm only saying we could mark the module as "upgrade-only" so that we don't release it together with kitchen-sink anymore

Comment (dnsl48) [reply .cow.json:23]: let's go with it as `not upgrade-only`. that's a safer default.

**Review by dnsl48 — APPROVED** [review]

**Pull request merged**